Index: .idea/webContexts.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"WebContextManager\">\r\n    <option name=\"state\">\r\n      <map>\r\n        <entry key=\"file://$PROJECT_DIR$/web/src/main/webapp/display.jsp\" value=\"file://$PROJECT_DIR$/web/src/main/webapp\" />\r\n        <entry key=\"file://$PROJECT_DIR$/web/src/main/webapp/registration.jsp\" value=\"file://$PROJECT_DIR$/web/src/main/webapp\" />\r\n        <entry key=\"file://$PROJECT_DIR$/web/src/main/webapp/secured/createTrip.jsp\" value=\"file://$PROJECT_DIR$/web/src/main/webapp/secured\" />\r\n        <entry key=\"file://$PROJECT_DIR$/web/src/main/webapp/secured/display.jsp\" value=\"file://$PROJECT_DIR$/web/src/main/webapp/secured\" />\r\n      </map>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/webContexts.xml b/.idea/webContexts.xml
--- a/.idea/webContexts.xml	(revision 988ceee4f564ebeda73af8fa6952437a8d78aa6c)
+++ b/.idea/webContexts.xml	(date 1637005972771)
@@ -7,6 +7,7 @@
         <entry key="file://$PROJECT_DIR$/web/src/main/webapp/registration.jsp" value="file://$PROJECT_DIR$/web/src/main/webapp" />
         <entry key="file://$PROJECT_DIR$/web/src/main/webapp/secured/createTrip.jsp" value="file://$PROJECT_DIR$/web/src/main/webapp/secured" />
         <entry key="file://$PROJECT_DIR$/web/src/main/webapp/secured/display.jsp" value="file://$PROJECT_DIR$/web/src/main/webapp/secured" />
+        <entry key="file://$PROJECT_DIR$/web/src/main/webapp/secured/displayCM.jsp" value="file://$PROJECT_DIR$/web/src/main/webapp/secured" />
       </map>
     </option>
   </component>
